src/registrar/abha-components/display-abha-card/display-abha-card.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="title info">
+    <h4 class="title">ABHA Card</h4>
```

Comment:
Pls add language implementation for heading, button names etc

Reply:
will be added in next PR